website/docs/reference/global-configs/logs.md

```diff
@@ -6,7 +6,7 @@ sidebar: "logs"
 
 ### Log Formatting
 
-The `LOG_FORMAT` config specifies how dbt's logs should be formatted. If the value of this config is `json`, dbt will output fully structured logs in <Term id="json" /> format; otherwise, it will output text-formatted logs that are sparser for the CLI and more detailed in `logs/dbt.log`.
+The `LOG_FORMAT` config specifies how dbt's logs should be formatted. If the value of this config is `json`, dbt will output fully structured logs in <Term id="json" /> format; otherwise, it will output text-formatted logs that are sparser for the CLI and more detailed in `logs/dbt.log`. The available options for the log format are `text`, `debug`, `json`, and `default`. The default is `default`. 
```

Comment:
@dbeatty10 may I ask you to clarify the difference (maybe provide a couple of examples) between `text` and `default`?